client/src/lib/utils/faucetRequest.ts

```diff
@@ -0,0 +1,39 @@
+import {DEMO_MODE, FAUCET_URL} from "./config";
+
+export async function request(address: string, recaptcha: string): Promise<string> {
+  if (DEMO_MODE) {
+    return boilerplateRequest(address, recaptcha);
+  }
+
+  return faucetRequest(address, recaptcha);
+}
+
+export async function faucetRequest(address: string, recaptcha: string): Promise<string> {
+  const body = {
+    address,
+    parachain_id: "1002",
```

Comment:
Shall we make this into a dropdown or something? (Maybe in next PR).
Even if we would only have 1 option right now, at least we would see where exactly the funds would go.

Reply:
Yeah, I agree.

I'll merge the current progress and I'll make a new PR with the id as a dropdown, that way the complexity is smaller and we can also brainstorm a bit the idea on how to do it.

Reply:
> that way the complexity is smaller and we can also brainstorm a bit the idea on how to do it

100% agree.